screens/Budget/components/missingBudget.tsx

```diff
@@ -52,17 +36,47 @@ const MissingBudget = (
         })
     })
 
-    function createCopy() {
-        alert("Your nearset budget is " + (closetBudget !== undefined ? MONTHS_ORDER[parseInt(closetBudget.split('-')[1])] : 'none'))
+    const [copyBudget] = useMutation<CopyBudgetMutation>(CopyBudgetDocument, {
```

Comment:
I would change this de-structuring to `[copyBudget, _]` to make it obvious that there is a second value that is returned but not used.